cli/src/test/java/denominator/cli/DenominatorTest.java

```diff
@@ -200,6 +202,31 @@ public void testConfigArgRoute53() {
   }
 
   @Test
+  public void testParseCertAndPrivateKeyInCredentials() {
+    ZoneList zoneList = new ZoneList() {
+      @Override
+      public Iterator<String> doRun(DNSApiManager mgr) {
+        assertThat(configPath).isEqualTo(getTestConfigPath("test-config-cert.yml"));
+        final Map mapCredentials = Map.class.cast(credentials);
```

Comment:
I think you can remove the instance ofs. Reason is that casts will also fail the test. less lines.

otherwise, looks alright as I don't know any way off top of head to do this any better :)